app/utils/results.py

```diff
@@ -223,3 +223,101 @@ def add_ip_card(html_soup: BeautifulSoup, ip: str) -> BeautifulSoup:
         # Inserting the element
         ref_element.insert_before(ip_tag)
     return html_soup
+
+
+def check_currency(response: str) -> dict:
+    """Check whether the results have currency conversion
+
+    Args:
+        response: Search query Result
+
+    Returns:
+        dict: Consists of currency names and values
+
+    """
+    soup = BeautifulSoup(response, 'html.parser')
+    currency_link = soup.find('a', {'href': 'https://g.co/gfd'})
+    if currency_link and currency_link.text.lower() == 'disclaimer':
```

Comment:
Also, does this still work when the search language is changed? If not, it's probably good enough to just check if the currency link exists.

Reply:
I will check it out